workers/config/src/main/kotlin/ConfigValidator.kt

```diff
@@ -20,12 +20,11 @@
 package org.eclipse.apoapsis.ortserver.workers.config
```

Comment:
Maybe add a few more details to the commit message, so that it is not necessary to look up the reference to understand the reason behind this change, e.g. something like "... to enable proper syntax highlighting in the validation script".
But I leave this up to you.